proto/error/error.proto

```diff
@@ -7,9 +7,11 @@ message StackFrame {
 }
 
 message Error {
-	int32 code = 1;
+	//int32 deprecated_code = 1;
```

Comment:
Can we remove these rather than commenting them? Also, given this is a breaking change to all the things anyway, I'd just number the proto fields sequentially.